lib/src/Uncertainty/Algorithm/Simulation/Wilks.cxx

```diff
@@ -86,6 +96,7 @@ Point Wilks::computeQuantileBound(const Scalar quantileLevel,
 {
   // Compute the needed sample size
   const UnsignedInteger size = ComputeSampleSize(quantileLevel, confidenceLevel, marginIndex);
+  std::cerr << "quantileLevel=" << quantileLevel << ", confidenceLevel=" << confidenceLevel << ", marginIndex=" << marginIndex << ", size=" << size << std::endl;
```

Comment:
drop the cerr

Reply:
I am rewriting the code, I found a numerical instability thanks to the plot_quantile_estimation_wilks.py example.

Reply:
ah ok, sorry